connector/connect/common/src/main/protobuf/spark/connect/base.proto

```diff
@@ -163,6 +164,14 @@ message AnalyzePlanRequest {
     // (Required) The logical plan to get a hashCode.
     Plan plan = 1;
   }
+
+  // Explains the expression based on extended is true or not.
+  message ExplainExpression {
+    // (Required) The expression to be analyzed.
+    Expression expr = 1;
+
+    bool extended = 2;
```

Comment:
Nit: please follow the style guide to indicate if this is required or optional and also document the semantics.

Reply:
Thank you! I forgot it.